x/emissions/proto/emissions/v4/query.proto

```diff
@@ -399,6 +399,21 @@ service QueryService {
     option (cosmos.query.v1.module_query_safe) = true;
     option (google.api.http).get = "/emissions/v4/next_churning_block_by_topic_id/{topic_id}";
   }
+
+  rpc GetActiveReputersForTopic(GetActiveReputersForTopicRequest) returns (GetActiveReputersForTopicResponse) {
+    option (cosmos.query.v1.module_query_safe) = true;
+    option (google.api.http).get = "/emissions/v4/active_reputers/{topic_id}";
+  }
+
+  rpc GetActiveForecastersForTopic(GetActiveForecastersForTopicRequest) returns (GetActiveForecastersForTopicResponse) {
+    option (cosmos.query.v1.module_query_safe) = true;
+    option (google.api.http).get = "/emissions/v4/active_forecasters/{topic_id}";
+  }
+
+  rpc GetActiveInferersForTopic(GetActiveInferersForTopicRequest) returns (GetActiveInferersForTopicResponse) {
+    option (cosmos.query.v1.module_query_safe) = true;
+    option (google.api.http).get = "/emissions/v4/active_inferers/{topic_id}";
+  }
 }
```

Comment:
Missing corresponding queries for the other new maps:

* LowestInfererScoreEma
* LowestForecasterScoreEma
* LowestReputerScoreEma
* LossBundles

Reply:
We already have queries for the first 3, please double-check. 

The latter, I don't think it is necessary because it's an ephemeral storage; it will be deleted when the reputer nonce is closed, but it is open to discussion. If we decide to not have it, I need to delete `GetWorkerLatestInferenceByTopicId`.